src/test/scala/com/github/dnvriend/serializer/avro4s/AvroSENoOldClassesTest.scala

```diff
@@ -68,172 +68,47 @@ class AvroSENoOldClassesTest extends AvroTestSpec {
   }
 
   "v1 to v2" should "decode with format" in {
-    val decoder = implicitly[Decoder[AvroCommand, MovieChangedV2]]
-    decoder.decode(AvroCommand(
-      Base64("BmZvb4wf"),
-      oldSchema =
-        """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 1,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string" },
-        |   { "name" : "year", "type" : "int" }
-        |  ]
-        |}
-      """.stripMargin.toSchema,
-      newSchema =
-        """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 2,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string" },
-        |   { "name" : "year", "type" : "int" },
-        |   { "name" : "director", "type" : "string", "default" : "unknown" }
-        |  ]
-        |}
-      """.stripMargin.toSchema
+    val decoder = implicitly[Decoder[AvroSchemaEvolution, MovieChangedV2]]
+    decoder.decode(AvroSchemaEvolution(
+      Base64("BmZvb4wf"), //MovieChangedV1("foo", 1990)
+      oldSchema = SchemaRegistry.movieChanged(1),
+      newSchema = SchemaRegistry.movieChanged(2)
     )) shouldBe MovieChangedV2("foo", 1990, "unknown")
   }
 
   "v2 to v3" should "decode with format" in {
-    val decoder = implicitly[Decoder[AvroCommand, MovieChangedV3]]
-    decoder.decode(AvroCommand(
-      Base64("BmZvb4wfBmJhcg=="),
-      oldSchema = """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 2,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string" },
-        |   { "name" : "year", "type" : "int" },
-        |   { "name" : "director", "type" : "string" }
-        |  ]
-        |}
-      """.stripMargin.toSchema,
-      newSchema = """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 3,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string"},
-        |   { "name" : "released_year", "type" : "int", "aliases" : ["year"] },
-        |   { "name" : "director", "type" : "string"}
-        |  ]
-        |}
-      """.stripMargin.toSchema
+    val decoder = implicitly[Decoder[AvroSchemaEvolution, MovieChangedV3]]
+    decoder.decode(AvroSchemaEvolution(
+      Base64("BmZvb4wfBmJhcg=="), //MovieChangedV2("foo", 1990, "bar")
```

Comment:
Added as comment the case class of previous version to make easier check difference between deserialized content